docs/src/examples/components/Menu/BestPractices/MenuBestPractices.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react'
+
+import ComponentBestPractices from 'docs/src/components/ComponentBestPractices'
+
+const doList = [
+  {
+    key: 'one',
+    content: 'Choose desired accessibility behavior depending on the use case.',
```

Comment:
Would it be helpful to add examples of what use cases would use which behaviors?

Reply:
@jurokapsiar, can you please provide some short examples?